2_process/src/process_data.R

```diff
@@ -1,20 +1,12 @@
-
-library(dplyr)
-library(readr)
-library(stringr)
-library(sbtools)
-library(whisker)
```

Comment:
Same goes for the library calls here - it's best practice to only load those libraries that are required for the functions defined here